### PR TITLE
Update metadata-rules.md

### PR DESCRIPTION
**Pull request by osimarkrogers**

### CODE REVIEW

File: Content_Portal/add-organize-data/attach-meaning/metadata-rules/metadata-rules.md [hunk 1]
```diff
@@ -4,9 +4,7 @@ uid: Metadatarules
 
 # Metadata rules
 
-The OCS Portal provides the ability to create metadata rules in a graphical user interface. Metadata rules assign defined metadata to all streams in a given namespace that match a defined stream name pattern. This provides a means of retrieving groups of similar streams for analytical purposes.
-
-You select a stream name to use as a name pattern and assign metadata to selected stream name parts, such as a plant location or device category. The resulting stream name pattern with assigned metadata parts defines a metadata rule.   
+Metadata rules assign defined metadata to all streams in a given namespace matching a defined stream name pattern. Use metadata rules to identify groups of similar streams for analytical purposes. You select a stream name to use as a name pattern and assign metadata to selected stream name parts, such as a plant location or device category. The resulting stream name pattern with assigned metadata parts defines a metadata rule.
```
Comment (afloresOSI) [inline Content_Portal/add-organize-data/attach-meaning/metadata-rules/metadata-rules.md:7]: This looks fine grammatically and spelling wise. 
I think the second sentence should actually be the first sentence. Maybe even expand it and combine it with the first sentence:  Use metadata rules to identify groups of similar streams for analytical purposes by assigning a naming pattern.

**Review by afloresOSI — APPROVED** [review]
Setting this to approved, but do consider my suggestions before merging.

File: Content_Portal/add-organize-data/attach-meaning/metadata-rules/metadata-rules.md (hunk 1 above)
Comment (osimarkrogers) [reply Content_Portal/add-organize-data/attach-meaning/metadata-rules/metadata-rules.md]: Rearranged per your suggestion. Thanks!

**Review by lasato-OSI — CHANGES_REQUESTED** [review]
Hi Mark,

This procedure needs to be cleaned up. Please make the changes to it and resubmit the PR and add me as a reviewer. If you need some help identifying what needs to be done, you could check with Joyce and see if she is able to do an edit. 

Thanks,
Laureen

**Comment by osimarkrogers** [comment]
> Hi Mark,
> 
> This procedure needs to be cleaned up. Please make the changes to it and resubmit the PR and add me as a reviewer. If you need some help identifying what needs to be done, you could check with Joyce and see if she is able to do an edit.
> 
> Thanks,
> Laureen

Hi

**Pull request closed without merging**